src/item_contents.cpp

```diff
@@ -249,11 +368,70 @@ bool pocket_favorite_callback::key( const input_context &, const input_event &ev
             }
         }
         return true;
-    } else if( input == 'x' ) {
-        const int pocket_num = menu->selected + 1;
+    } else if( action == "FAV_CLEAR" ) {
         if( query_yn( _( "Are you sure you want to clear settings for pocket %d?" ), pocket_num ) ) {
             selected_pocket->settings.clear();
         }
+    } else if( action == "FAV_CONTEXT_MENU" ) {
+        uilist cmenu( string_format( "Action to take on this pocket" ), {} );
```

Comment:
```suggestion
        uilist cmenu( _( "Action to take on this pocket" ), {} );
```
Woops, this one was my fault :P